docs/docs/concepts/foundation/communication/public_private_calls/main.md

```diff
@@ -4,7 +4,7 @@ title: Private <--> Public execution
 
 import Image from "@theme/IdealImage";
 
-import Disclaimer from "../../../misc/common/\_disclaimer.mdx";
+import Disclaimer from "../../../../misc/common/\_disclaimer.mdx";
```

Comment:
this file btw talks about blacklists. Can you update that paragraph to tease the slow updates tree and link to your new page?

Also - feel free to search for other references to that term and tease slow updates there and hyperlink your docs